README.md

```diff
@@ -30,6 +30,10 @@ The request body is a YAML (JSON) encoded flavor description:
 customization:
     extraKernelArgs: # optional
         - vga=791
+    systemExtensions: # optional
+      officialExtensions: # optional
```

Comment:
maybe `trustedExtensions` well probably doesn't matter since many extensions are community contributed :shrug:

Reply:
I think `official` is like blessed by us... basically part of the extensions catalog, could be not built by us in theory